adapted-articles/S/SpiritofGod.xml

```diff
@@ -5,36 +5,36 @@
 <p>In the earlier stages of Hebrew thought, ecstatic experience was seen as the direct effect of divine power. This was true even when the ecstasy was recognized as evil in character, as in the case of Saul’s seizure by the Spirit (<a href="?bref=1Sam.16.14-16">1 Sm 16:14-16</a>). A spirit from God could be for evil as well as for good (see <a href="?bref=Judg.9.23">Jgs 9:23</a>; <a href="?bref=1Kgs.22.19-23">1 Kgs 22:19-23</a>).</p>
 <p class="h3">In the Writings of the Prophets</p>
-<p class="fl">For Isaiah, the spirit was that which characterized God and distinguished him and his actions from human affairs (<a href="?bref=Isa.31.3">Is 31:3</a>). Later, the adjective “holy” appeared as that which distinguished the Spirit of God from any other spirit, human or divine (<a href="?bref=Ps.51.11">Ps 51:11</a>; <a href="?bref=Isa.63.10-11">Is 63:10-11</a>).</p>
-<p>The problem of false prophecy emphasized the danger of assuming that every message delivered in ecstasy was the word of the Lord. Thus, tests of prophecy evaluated the content of the message delivered or the character of the prophet’s life, not the degree or quality of inspiration (see <a href="?bref=Deut.13.1-5">Dt 13:1-5</a>; <a href="?bref=Deut.18.22">18:22</a>; <a href="?bref=Jer.23.14">Jer 23:14</a>; <a href="?bref=Mic.3.5">Mi 3:5</a>). This sense of a need to discriminate between true and false inspiration and to distinguish the word of God from the merely ecstatic oracle may help to explain the otherwise puzzling reluctance of the major eighth- and seventh-century <span class="era">BC</span> prophets to attribute their inspiration to the Spirit.</p>
+<p class="fl">For Isaiah, the spirit was that which characterized God and distinguished him and his actions from human affairs (<a href="?bref=Isa.31.3">Is 31:3</a>). Later, the adjective “holy” appeared to distinguish the Spirit of God from any other spirit, human or divine (<a href="?bref=Ps.51.11">Ps 51:11</a>; <a href="?bref=Isa.63.10-11">Is 63:10-11</a>).</p>
+<p>The problem of false prophecy emphasized the danger of assuming that every message delivered in ecstasy was the word of the Lord. Therefore, the content of the message delivered was tested, as well as the character of the prophet’s life, but not the degree or quality of inspiration (see <a href="?bref=Deut.13.1-5">Dt 13:1-5</a>; <a href="?bref=Deut.18.22">18:22</a>; <a href="?bref=Jer.23.14">Jer 23:14</a>; <a href="?bref=Mic.3.5">Mi 3:5</a>). This sense of a need to differentiate between true and false inspiration and to distinguish the word of God from the any ecstatic message supposed from God may help to explain the otherwise puzzling reluctance of the major eighth- and seventh-century <span class="era">BC</span> prophets to attribute their inspiration to the Spirit.</p>
```

Comment:
"supposedly from God" or "supposed to be from God"

Reply:
“from any ecstatic message supposedly from God” works for me.